configurations/ms-ad-ldap-authorization-and-authentication.yaml

```diff
@@ -0,0 +1,9 @@
+use_ms_ad_ldap: true
+authenticator: 'com.scylladb.auth.SaslauthdAuthenticator'
+prepare_saslauthd: true
+use_saslauthd_authenticator: true
+authenticator_user: 'cassandra'
+authenticator_password: 'cassandra'
+
+authorizer: 'CassandraAuthorizer'
+#use_ldap_authorization: true
```

Comment:
remove dead code

Reply:
also, this file is exactly the same as the previous one, so, why do we need 2 files?

Reply:
I also don't understand why we have two identical files...

Reply:
`use_ms_ad_ldap` actually means  use ms_ad_ldap for authorization in Fabio's PR, it `isn't` an option to switch between docker openldap and ms ad ldap.

I thought we should enable `use_ldap_authorization` and `use_ms_ad_ldap' together for enable authorization of ms_ad ldap.

It's not good to add a new parameter to only enable ms_ad_ldap authentication (eg: `use_ms_ad_ldap_authentication`)

@fgelcer any idear?

Reply:
`use_ms_ad_ldap` will enable `ldap_authorization` with the LDAP back end being the MS AD server.
in case of authorization, what can be done is that we set both flags for authorization (`use_ldap_authorization` and `use_ms_ad_ldap`) and then change the code in the `Setup` to not create the docker, and get the configuration to `scylla.yaml` from the right place.

Reply:
> `use_ms_ad_ldap` will enable `ldap_authorization` with the LDAP back end being the MS AD server.
> in case of authorization, what can be done is that we set both flags for authorization (`use_ldap_authorization` and `use_ms_ad_ldap`) and 

> then change the code in the `Setup` to not create the docker, and get the configuration to `scylla.yaml` from the right place.

It's already implemented in this PR.